.gitmodules

```diff
@@ -1,3 +1,21 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/solidity-bits"]
+	path = lib/solidity-bits
+	url = https://github.com/estarriolvetch/solidity-bits
+[submodule "lib/solidity-bytes-utils"]
+	path = lib/solidity-bytes-utils
+	url = https://github.com/GNSPS/solidity-bytes-utils
```

Comment:
Should we be scrutinizing these deps?

Reply:
Those are brought in by the ERC1155, ERC 721Permit, and ERC721Psi code. I have brought those in so that the forge build would work.

Both of them have been around for a long time. However, we aren't bringing in a specific version.

I think we should address improvements related to them in a separate PR.